Monika After Story/game/script-topics.rpy

```diff
@@ -5079,6 +5079,79 @@ label monika_beach:
     m 1k "Don't get too excited though when you see it, Ehehehe~"
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_trick",category=['club members'],prompt="[player]'s Second Choice",random=True))
+
+label monika_trick:
+    m 1c "Say [player], I have a question..."
+    m 3o "I hope I don't sound insecure when I say this..."
+    m 3m "I know you love me and only me but... if you had to choose one of the other club members..."
+    show screen mas_background_timed_jump(10, "monika_trick_2")
+    menu:
+        m "Who would you choose?"
+        "Yuri":
```

Comment:
You need to hide the timed jump screen after a user selects a choice to prevent the screen from jumping.
this is because the screen runs in the background.

I'd put the `hide screen` lines in the labels below, since both `monika_trick` and `monika_trick_2` go to the same labels.